.circleci/config.yml

```diff
@@ -337,6 +385,25 @@ references:
               # Demisto Marketplace refers to Server Master
               SERVER_URL=$(jq -r 'select(.[].Role == "Demisto Marketplace") | .[].InstanceDNS' ./env_results.json)
               python3 ./Utils/comment_on_pr.py -p $PULL_REQUEST_NUMBER -c "Instance is ready. Server link: https://$SERVER_URL, Build link: $CIRCLE_BUILD_URL"
+
+  upload_entities_to_cortex_xsoar: &upload_entities_to_cortex_xsoar
+    run:
+        name: Upload entities to Cortex XSOAR
+        command: |
+          demisto-sdk upload -i Packs/HelloWorld/Integrations/ --insecure
+          demisto-sdk upload -i Packs/HelloWorld/TestPlaybooks/playbook-HelloWorld-Test.yml --insecure
+          # TODO: After fixing the demisto-sdk upload, upload from here instead of Tests/sdknightly/create_entities_for_nightly_sdk.py
```

Comment:
is there issue for this?

Reply:
Yeah, I opened one and @ShahafBenYakir said there isn't one. reopening https://github.com/demisto/etc/issues/29578

Reply:
This is hot fake news is spread...